sigs/cncf-sigs.md

```diff
@@ -0,0 +1,252 @@
+
+#### As Trusted Expert Advisors to the TOC
+
+    * Perform technical due diligence on new and graduating projects, and advise TOC on findings.
+
+    * Be involved with, or periodically check in with projects in their area, and advise TOC on health, status and proposed actions (if any) as necessary or on request.
+
+#### SIG Charter:
+
+    * formally reviewed annually, and approved by the TOC.  The charter must clearly articulate:
+
+        * what is in and out of scope of the SIG
+
+        * whether and how it overlaps and interfaces with other CNCF SIG’s or other relevant groups.
+
+        * how it operates and is governed, and specifically whether and how it deviates from standard SIG operating guidelines provided by the TOC.  Deviation from these guidelines is discouraged, unless there are good and well-documented reasons for such divergence, approved by the TOC. 
+
+See [Example Responsibilities of a CNCF SIG](https://docs.google.com/document/d/1L9dJl5aBFnN5KEf82J689FY0UtnUawnt9ooCq8SkO_w/edit?usp=sharing).
+
+## Operating Model
+
+*Important*
+
+*Each SIG is supported by a named member of the CNCF executive staff who is accountable for liaison with the CNCF Exec, plus communication and performance of the SIG, with quarterly and annual reporting to GB & TOC. *** **
+
+As a starting point let’s be inspired by CNCF OSS Projects and by K8S SIGs.  That means minimal viable governance and community-based organisation.
+
+### SIG Formation, Leadership and Membership Composition
+
+1. SIGs are formed by the TOC.  Initial SIGs are listed below, and will be adapted over time as required.  If members of the community believe that additional SIGs are desired, they should propose these to the TOC, with clear justification, and ideally volunteers to lead the SIG. The TOC wishes to have the smallest viable number of SIGs, and for all of them to be highly effective (as opposed to a "SIG sprawl" with large numbers of relatively ineffective SIGS).
+
+2. SIG has three co-chairs, who are TOC Contributors and recognised as unbiased experts in that area.
+
+3. SIG has one TOC liaison who is a voting member of the TOC acting as non-exec chair on occasions when TOC input is deemed necessary by the TOC or the SIG chairs
+
+4. SIG has multiple tech leads who are recognised as unbiased experts in the SIG area, and are recognised leaders of projects in the SIG’s area.  The reason for having separate chair and tech lead roles is to allow responsibility for primarily administrative functions to be separated from deep technical functions and associated time commitments and skill sets.  Where appropriate, an individual may perform both roles (see below).
+
+5. Thought and interest diversity is strongly encouraged within SIGs.  To this end, a supermajority (⅔ or more) of chairs or tech leads from a single group of companies, market segment, etc will be actively discouraged by the TOC.
```

Comment:
Do you mean 2/3 of chairs and 2/3 of tech leads, or 2/3 of chairs+tech leads?  It's not clear. If we mean 2/3 of chairs, then it might be easier to just say "no more than one chair from the same company" since that's what it means.

Reply:
@duglin I avoided using the "no more than one" wording, in case the number of chairs (or tech leads) changes. The key point is to avoid a super-majority, not to avoid more than one, necessarily.  I'll change the wording to make it clearer that 2/3 or more chairs, and 2/3 or more tech leads (i.e. each individually) will be actively discouraged.